nautobot/extras/utils.py

```diff
@@ -391,6 +397,19 @@ def get_worker_count(request=None, queue=None):
     return celery_queues.get(queue, 0)
 
 
+def get_job_queue_worker_count(request=None, job_queue=None):
```

Comment:
If this is going to stay celery-specific let's give it a name that indicates such. If it isn't, let's add appropriate TODOs and/or refactor it so that it doesn't call `get_celery_queues` if the given `job_queue` is not a Celery queue.